doc/users/issue_commands.rst

```diff
@@ -19,17 +19,15 @@ For example, to record a message at a specific id with the :ref:`developers/exam
     HTTP/1.1 200 OK
     content-length: 5
     content-type: application/json
-    x-ccf-commit: 23
-    x-ccf-global-commit: 22
-    x-ccf-term: 2
+    x-ccf-tx-seqno: 23
+    x-ccf-tx-view: 2
 
     true
 
 The HTTP response some CCF commit information in the headers:
```

Comment:
Unchanged, but spotted here: Should say "response **contains** some"

Reply:
Yes, thank you!